branchff

```diff
@@ -116,6 +116,16 @@ logrun mkdir -p $WORKDIR
 gitlib::sync_repo $K8S_GITHUB_SSH $TREE_ROOT || common::exit 1 "Exiting..."
 logrun cd $TREE_ROOT
 
+# Is the master branch in a reasonable state to fast forward over the branch?
+GIT_DESCRIBE=$(git describe)
+if [[ ! $GIT_DESCRIBE =~ ^v${RELEASE_BRANCH/release-/} ]]; then
```

Comment:
ah, here's a method that should work: compare `git describe --abbrev=0 master` against `git describe --abbrev=0 $(git merge-base release-1.7 master)`. if they match we're OK; if not, throw an error.